docs/src/further_topics/ugrid/operations.rst

```diff
@@ -0,0 +1,953 @@
+                    <AuxCoord: longitude / (unknown)  <lazy>  shape(2,)>
+                    <AuxCoord: latitude / (unknown)  <lazy>  shape(2,)>
+            long_name: 'my_mesh'
+            var_name: 'my_mesh'
+
+Plotting
+--------
+.. |tagline: plotting| replace:: |different| - plot with GeoVista
+
+.. rubric:: |tagline: plotting|
+
+The Cartopy-Matplotlib combination is not optimised for displaying the high
+number of irregular shapes associated with meshes. Thankfully mesh
+visualisation is already popular in many other fields (e.g. CGI, gaming,
+SEM microscopy), so there is a wealth of tooling available, which
+:ref:`ugrid geovista` harnesses for cartographic plotting.
+
+GeoVista's default behaviour is to convert lat-lon information into full XYZ
+coordinates so the data is visualised on the surface of a 3D globe. The plots
+are interactive by default, so it's easy to explore the data in detail.
+
+2D projections have also been demonstrated in proofs of concept, and will
+be added to API in the near future.
+
+This first example uses GeoVista to plot the ``face_cube`` that we created
+earlier:
+
+.. dropdown:: :opticon:`code`
+
+    .. code-block:: python
+
+        >>> from geovista import GeoPlotter, Transform
+        >>> from geovista.common import to_xyz
+
+
+        # We'll re-use this to plot some real global data later.
+        >>> def cube_faces_to_polydata(cube):
+        ...     lons, lats = cube.mesh.node_coords
+        ...     face_node = cube.mesh.face_node_connectivity
+        ...     indices = face_node.indices_by_location()
+        ...
+        ...     mesh = Transform.from_unstructured(
+        ...         lons.points,
+        ...         lats.points,
+        ...         indices,
+        ...         data=cube.data,
+        ...         name=f"{cube.name()} / {cube.units}",
+        ...         start_index=face_node.start_index,
+        ...     )
+        ...     return mesh
+
+        >>> print(face_cube)
+        face_data / (K)                     (-- : 2; height: 3)
+            Dimension coordinates:
+                height                          -          x
+            Mesh coordinates:
+                latitude                        x          -
+                longitude                       x          -
+            Attributes:
+                Conventions                 CF-1.7
+
+        # Convert our mesh+data to a PolyData object.
+        # Just plotting a single height level.
+        >>> face_polydata = cube_faces_to_polydata(face_cube[:, 0])
+        >>> print(face_polydata)
+        PolyData (0x7ff4861ff4c0)
+          N Cells:	2
+          N Points:	5
+          X Bounds:	9.903e-01, 1.000e+00
+          Y Bounds:	0.000e+00, 1.392e-01
+          Z Bounds:	6.123e-17, 5.234e-02
+          N Arrays:	2
+
+        # Create the GeoVista plotter and add our mesh+data to it.
+        >>> my_plotter = GeoPlotter()
+        >>> my_plotter.add_coastlines(color="black")
```

Comment:
```suggestion
        >>> my_plotter.add_coastlines()
```
@trexfeathers The associated rendered image has `white` coastlines (default), and yet you specify `black` here in the example... I guess the code and image have got out of sync.

Reply:
Tut tut @trexfeathers 

4c2b19a76783d5dd5ca6d1af91e1887bb59b5058